### DISCUSSION TITLE
I need help guys

### DISCUSSION THREAD

**Discussion by spinks007** [ZerBea/hcxdumptool#474] [Q&A]
hcxdumptool -o test.hc22000 --essidlist dumpfile.pcapngls
hcxdumptool: invalid option -- 'o'
hcxdumptool 6.3.4-43-g5d26098 (C) 2024 by ZeroBeat
This is a penetration testing tool!
It is made to detect vulnerabilities in your NETWORK mercilessly!

**Comment by ZerBea** [comment]
Since version 6.3.0 options have changed as mentioned in changelog (03.05.2023).
To show an overview use -h and --help
and read:
https://github.com/ZerBea/hcxdumptool/discussions/423
https://github.com/ZerBea/hcxdumptool/discussions/388
https://github.com/ZerBea/hcxdumptool/discussions/343

Filtering is now done via a Berkeley Packet Filter directly in kernel space:
https://github.com/ZerBea/hcxdumptool/discussions/420
